### PR TITLE
Features 1330 client error handling

### PR DESCRIPTION
**Pull request by yecli**
- Logout user if he deletes himself
- Added client error handling for import/export
image

### CODE REVIEW

File: VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js [hunk 1]
```diff
@@ -46,7 +46,12 @@ function ($scope, accounts, dialogService, uiGridHelper, bladeNavigationService,
                 if (remove) {
                     bladeNavigationService.closeChildrenBlades(blade, function () {
                         var itemIds = _.pluck(selection, 'userName');
-                        accounts.remove({ names: itemIds }, blade.refresh);
+                        accounts.remove({ names: itemIds }, function () {
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js]: I don't think that is good idea to allow user remove self account from system, could we prevent this action?

File: VirtoCommerce.Platform.Web/Scripts/app/exportImport/blades/export-main.js [hunk 2]
```diff
@@ -10,6 +10,8 @@
         exportImportResourse.getNewExportManifest(function (data) {
             $scope.exportRequest.exportManifest = data;
             blade.isLoading = false;
+        }, function (error) {
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Scripts/app/exportImport/blades/export-main.js]: Could you check why the common platform error handling doesn't work for this actions?
https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Scripts/app/app.js#L59

File: VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js (hunk 1 above)
Comment (yecli) [reply VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js]: Yes, we could do this. But in many systems removing self account is a legal action (like Github). Maybe better to add confirmation that user is trying to delete himself (or add that info to initial confirmation on Delete action)?

Comment (tatarincev) [reply VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js]: Yes, confirmation will be a good choice.

Comment (yecli) [reply VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js]: Decided to restrict possibility of self deletion on server side.

File: VirtoCommerce.Platform.Web/Scripts/app/exportImport/blades/export-main.js (hunk 2 above)
Comment (yecli) [reply VirtoCommerce.Platform.Web/Scripts/app/exportImport/blades/export-main.js]: It worked, in fact, so reverted redundant handlers. Also changed error handler for FileUploader to produce error with non empty details.

**Pull request merged**